content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -26,87 +26,211 @@ Tracer settings can be configured with the following environment variables:
 ### Tagging
 
 `DD_ENV`
-: Set an application's environment (for example, `prod`, `pre-prod`, and `stage`). Defaults to the environment configured in the Datadog Agent.
+: **Configuration**: `env`<br>
+**Default**: Defaults to the environment configured in the Datadog Agent<br>
```

Comment:
Is the `Defaults to` part consistent with our other languages? Seems a bit redundant to me. 😅